ennMethods.rb

```diff
@@ -0,0 +1,152 @@
+# Module
+module Enumerable
+  #my_each: clone of each
+  def my_each
+    for x in self do
+      yield(x)
+    end
+    return self
```

Comment:
[Optional] Not required to complete this project.  Explicitly declaring return is not necessary. Ruby methods by default return the last code written.

Reply:
Fixed the problem on my_count.
Thanks for the suggestion